src/api/app/views/webui2/webui/projects/rebuild_times/show.html.haml

```diff
@@ -0,0 +1,35 @@
+:ruby
+  content_for(:content_for_head, javascript_include_tag('webui2/cm2/index-prjconf'))
+  @pagetitle = "Rebuild Time for Project #{@project}"
+  @metarobots = 'noindex'
+  p1 = @longestpaths[0].reverse
+  p2 = @longestpaths[1].reverse
+  p3 = @longestpaths[2].reverse
+
+.card
+  = render(partial: 'webui/project/tabs', locals: { project: @project })
+  .card-body
+    %h3
+      Rebuild Time: #{distance_of_time_in_words(Time.now, Time.now - @rebuildtime)} (#{format_seconds(@rebuildtime)})
+    %p
+      %img.rebuildgraph{ alt: 'The image could not be loaded. This should be a rebuild time graph.',
+        src: project_rebuild_time_png_path(project: @project, key: @pngkey) }/
+    %p
+      Note that the above graph is based on real dependencies and real build times, but it is only a simulation of what
+      may happen when there is no other trouble and with a fixed set of build jobs associated to the project (usually not the
+      case). To see what happens with less or more build hosts, play with these numbers:
+    = form_tag(project_rebuild_time_path(project: @project.name), method: :get, class: 'form-inline mb-2') do
```

Comment:
I think `.name` shouldn't be needed here

Reply:
> I think `.name` shouldn't be needed here

Right. Removed.